### PR TITLE
Bug 1221491 - Log the feature that could not be started/stopped

### PR DESCRIPTION
**Pull request by gregarndt**
This should now output an error such as:
[taskcluster:error] Task was aborted because states could not be created successfully. Error linking balrogVPNProxy : Insufficient scopes to use 'balrogVPNProxy' feature.  Try adding docker-worker:feature:balrogVPNProxy to the .scopes array.

### CODE REVIEW

**Comment by gregarndt** [comment]
@walac  Could you also take a look at this one?  Thanks!

**Comment by walac** [comment]
Yes, sure.

File: lib/features/dind.js [hunk 1]
```diff
@@ -76,7 +77,7 @@ export default class DockerInDocker {
     };
   }
 
-  killed(task) {
+  async killed(task) {
     task.runtime.gc.removeContainer(this.container.id);
```
Comment (walac) [inline lib/features/dind.js]: A see the `async` keyword but none `await` added, is this fixing some mistake?

Comment (gregarndt) [inline lib/features/dind.js]: This is so the method is treated as a promise for all features so we can call .then() in states.js . Can't call then() on a regular method

File: lib/task.js [hunk 2]
```diff
@@ -702,13 +702,13 @@ export class Task {
...
       catch (e) {
-        debug(e.stack);
         return await this.abortRun(
           'states_failed',
           this.fmtErrorLog(
             'Task was aborted because states could not be started ' +
-            `successfully. ${e.stack || e}`
-          )
+            `successfully. ${e}`
+          ),
```
Comment (walac) [inline lib/task.js]: Isn't this supposed to be `e.message`? (or the code above be simply `e`?)

Comment (gregarndt) [inline lib/task.js]: The line above I think I want to be e.message. This e is passed to abort run and will be logged to paper trail. I want the whole error passed there

**Pull request merged**